helm/charts/kratos/templates/hpa-v1.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (.Values.autoscaling.enabled) (not (.Capabilities.APIVersions.Has "autoscaling/v2beta2")) }}
+{{- if and (.Values.autoscaling.enabled) (not (.Capabilities.APIVersions.Has "autoscaling/v2")) }}
```

Comment:
this file was only for the transition period, when hpa was migrating from v1 to v2, i think we can delete it now :)